arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,936 @@
+      "post": {
+        "tags": [
+          "WorkspaceCollections"
+        ],
+        "operationId": "WorkspaceCollections_migrate",
+        "description": "Migrates an existing Power BI Workspace Collection to a different resource group and/or subscription.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "Azure resource group",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Workspace migration request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MigrateWorkspaceCollectionRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Migration completed successfully"
+          },
+          "default": {
+            "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Error": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorDetail"
+          }
+        }
+      }
+    },
+    "ErrorDetail": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        }
+      }
+    },
+    "WorkspaceCollectionList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WorkspaceCollection"
+          }
+        }
+      }
+    },
+    "WorkspaceList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Workspace"
+          }
+        }
+      }
+    },
+    "OperationList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        }
+      }
+    },
+    "AzureSku": {
+      "type": "object",
+      "properties": {
```

Comment:
@mattmazzola - The CI is failing because required is marked incorrectly.
```
1) Azure Swagger Schema Validation /home/travis/build/Azure/azure-rest-api-specs/arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json should be valid Swagger:

320     Uncaught Error: Schema validation failed: [

321	{

322		"code": "INVALID_TYPE",

323		"params": [

324			"array",

325			"boolean"

326		],

327		"message": "Expected type array but found type boolean",

328		"path": "#/definitions/AzureSku/properties/tier/required",

329		"schemaId": "https://raw.githubusercontent.com/Azure/autorest/master/schema/swagger-extensions.json#"

330	}

331]

332      at test/test.js:64:19

333      at fs.js:334:14

334      at FSReqWrap.oncomplete (fs.js:95:15)

335

```
required is an array. Please take a look at this example https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2016-01-01/swagger/storage.json#L426 for defining required properties of a model definition.